example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
+class m2mprint:
+    def __unicode__(self):
```

Comment:
@rohithasrk I'm looking now.. wait a minute and I'll come back to you regarding this